push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/token/TokenLoader.java

```diff
@@ -75,14 +74,6 @@
     @DispatchToQueue
     private Event<MessageHolderWithVariants> nextBatchEvent;
 
-    @Inject
-    @DispatchToQueue
-    private Event<BatchLoadedEvent> batchLoaded;
```

Comment:
If the events are not used anymore in the project, can we delete the classes BatchLoadedEvent and AllBatchesLoadedEvent?